trading/trading.go

```diff
@@ -67,10 +67,12 @@ func (p AbstractPortfolio) Value(prices Prices) float64 {
 
 // TODO: include enough data for other types of orders and shorts.
 type Order struct {
-	StopPrice  float64
-	LimitPrice float64
-	Volume     int
+	Type   string
```

Comment:
Would it be easy to make this an enum/iota?

Reply:
Yup. Super Easy